spec/unit/installer/xcode/pods_project_generator/aggregate_target_installer_spec.rb

```diff
@@ -1,7 +1,7 @@
-require File.expand_path('../../../../spec_helper', __FILE__)
+require File.expand_path('../../../../../spec_helper', __FILE__)
 
 module Pod
-  describe Installer::AggregateTargetInstaller do
+  describe AggregateTargetInstaller = Installer::Xcode::PodsProjectGenerator::AggregateTargetInstaller do
```

Comment:
please don't do this since it actually creates new constants in the Pod module that could cause tests to behave differently than when running `pod`